pkg/api/v1/config.go

```
@@ -84,3 +84,25 @@ func init() {
 		}
 	}
 }
+
+// RdsCa returns the filename for a temporary file
+func (a AppConfig) RdsCa() string {
+	dir, err := ioutil.TempDir("", "rdsca")
+	if err != nil {
+		fmt.Println(err)
```

Should we return the error instead?